tests/Core.Tests/VerbPatternDetectorTests.fs

```diff
@@ -78,7 +79,10 @@ let ``Detects pattern is not čistit`` word =
 let ``Gets pattern class E`` verb pattern =
     verb
     |> getPatternClassE
-    |> equals (Some pattern)
+    |> Option.get
+    |> string
+    |> fun s -> s.ToLower()
+    |> equals pattern
```

Comment:
I know this looks lame but `InlineData` in `Xunit.Theory` has to be constant so I did this crude adjustment for now and will soon rewrite the tests to thoroughly check each subpattern.

Reply:
I think you can put this into a helper method. e.g.
```fsharp
let checkPattern classGetter verb pattern =
  verb
    |> classGetter
    |> Option.get
    |> string
    |> fun s -> s.ToLower()
    |> equals pattern

let ``Gets pattern class E`` =
  checkPattern getPatternClassE
```

Reply:
Well, class getters have different signatures, e.g. `string -> ConjugationPatternClassE` for `getPatternClassE` and `string -> ConjugationPatternClassNE` for `getPatternClassNE`, so it cannot be generalized just like this.

But that's basically a signal for writing proper tests against every subpattern which is to be done.

Reply:
```fsharp
let checkPattern (classGetter: string -> 'a) verb (pattern: 'a) =
  verb
    |> classGetter
    |> Option.get
    |> string
    |> fun s -> s.ToLower()
    |> equals pattern
```
try this then

Reply:
Well if you really insist... see updated version :D

Reply:
Perfect! Thanks :)